<launch>
    <arg name="real_robot" default="false"/>
    <arg name="record" default="false"/>
    <include file="$(find voronoi_hsi)/launch/map_server.launch"/>
    <node name="image_view_tesselation" pkg="image_view" type="image_view" args="image:=/voronoi/tesselation" respawn="true" output = "screen"/>
    <node name="h_func_plot" pkg="rqt_plot" type="rqt_plot" args="/voronoi/h_func" />
    <group if="$(arg real_robot)">
        <include file="$(find voronoi_hsi)/launch/ar_track.launch"/>
    </group>
    <group if="$(arg record)">
        <node name="rosbag" pkg="rosbag" type="record" args="--all -o $(find voronoi_hsi)/bag/" output = "screen"/>
    </group>
</launch>
